projects/colorpicker/package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "@skyux/colorpicker",
+  "version": "5.0.0-beta.0",
+  "peerDependencies": {
+    "@angular/common": "^12.2.0",
+    "@angular/core": "^12.2.0",
+    "@angular/forms": "^9.1.9 || ^12",
```

Comment:
Probably doesn't matter (not sure?), but we should make this entry match the other Angular ones.